src/components/Elements/Pips.js

```diff
@@ -8,17 +8,26 @@ class Pips extends Component {
       currentIndex,
     } = this.props;
 
+    let pips = [];
+
+    for(let index = 0; index < count; index++) {
```

Comment:
Shall we add this to the lint settings? Wondering if we should extend from stricter preset too like airbnb?